master/internal/api_runs_intg_test.go

```diff
@@ -0,0 +1,63 @@
+//go:build integration
+// +build integration
+
+package internal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/master/pkg/ptrs"
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
+)
+
+func TestSearchRuns(t *testing.T) {
```

Comment:
can we test that every field can be filtered on?

Reply:
Added tests for all filter types (isEmpty, contains, etc...) as well as test for standard columns & hyperparameters